packages/babel-plugin-minify-dead-code-elimination/src/index.js

```diff
@@ -6,6 +6,11 @@ module.exports = ({ types: t, traverse }) => {
   const removeOrVoid = require("babel-helper-remove-or-void")(t);
   const shouldRevisit = Symbol("shouldRevisit");
 
+  // this is used for tracking fn params that can be removed
+  // as traversal takes place from left and
+  // unused params can be removed only on the right
+  const markForRemoval = Symbol("markForRemoval");
```

Comment:
hmmm. IMO, depends on context, 

at the place of assignment, mark this for removal,
at the place of verifying, is this marked for removal,

Not sure. IMO, either way should be fine.